code/src/Components/Thoughts.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react'
+import { HeartButton } from 'Components/HeartButton'
+import TimeAgo from 'timeago-react'
+
+import './thoughts.css'
+
+export const Thoughts = () => {
+  const [thoughts, setThoughts] = useState([])
+
+  const MESSAGES_URL = 'https://technigo-thoughts.herokuapp.com/'
+
+  useEffect(() => {
+    fetch(MESSAGES_URL,{method: 'GET'})    
+    .then(res => res.json())
+    .then(json => setThoughts(json))
+  },[])
+
+  const onThoughtLiked = (likedThoughtId) => {
```

Comment:
In case you didn't know - you don't need `()` around the argument if it's only one
```suggestion
  const onThoughtLiked = likedThoughtId => {
```